readme.md

```diff
@@ -1,2 +1,75 @@
 React Material Icons
 ====================
+
+This package includes all [Material Icons](https://material.io/resources/icons/?style=baseline) as reusable React components
+
+## Table of content
+- [Usage](#usage)
+- [Generate Component](#generate-component)
+- [Build & Publish](#build--publish)
+- [Information](#information)
+
+## Usage
+
+In order to use this package you have to install it through npm
+```bash
+npm install @aboutbits/react-material-icons
+```
+
+After you successfully installed the package you can simply use the icons in you React application like this
+```jsx
+import React from 'react'
+
+const MyCommponent = () => {
+    return (
+        <IconCached />
+    );   
+}
+``` 
+
+SVG related parameters like height and width can be passed as props
+
+## Generate Component 
+To generate the components, simply run the following command
+```bash
+npm run build
+```
+
+or
+
+```bash
+npm run bootstrapping
+```
+
+## Build & Publish
+
+To publish the package commit all changes and push them to master. Then run one of the following commands locally:
+```bash
+npm version patch
+npm version minor
+npm version major
+```
+
+To remove all generated files run the clean script
```

Comment:
Should the user be able to clean the generated files? Or should they be automatically be deleted as first step of the generation? For example, first delete the src folder and then regenerate the files in there.

The clean task that you mention here, is to delete the dist folder. And the dist folder is just required for publishing.